platform.go

```diff
@@ -147,14 +145,18 @@ func NewBindingsFromPath(path string) (Bindings, error) {
 	return bindings, nil
 }
 
-// NewBindingsForBuild creates a new bindings from all the bindings at the path defined by $SERVICE_BINDING_ROOT.
-// If isn't defined, bindings are read from <platform>/bindings, the default
-// path defined in the CNB Binding extension specification.
-func NewBindingsForBuild(platformDir string) (Bindings, error) {
+// NewBindings creates a new bindings from all the bindings at the path defined by $SERVICE_BINDING_ROOT.
+// If that isn't defined, bindings are read from <platform>/bindings.
+// If that isn't defined, the specified platform path will be used
+func NewBindings(platformDir string) (Bindings, error) {
 	if path, ok := os.LookupEnv(EnvServiceBindings); ok {
 		return NewBindingsFromPath(path)
 	}
 
+	if path, ok := os.LookupEnv("CNB_PLATFORM_DIR"); ok {
```

Comment:
Should we make this env var a public const as well?

Reply:
Yes, I can definitely do that.